src/status_im/chat/views/actions.cljs

```diff
@@ -1,59 +1,55 @@
 (ns status-im.chat.views.actions
-  (:require-macros [status-im.utils.views :refer [defview letsubs]])
-  (:require [re-frame.core :refer [subscribe dispatch]]
-            [clojure.string :as s]
-            [status-im.components.react :refer [view
-                                                animated-view
-                                                text
-                                                icon
-                                                touchable-highlight]]
-            [status-im.components.chat-icon.screen :refer [chat-icon-view-menu-item]]
-            [status-im.chat.styles.screen :as st]
-            [status-im.i18n :refer [label label-pluralize]]
-            [status-im.utils.platform :refer [platform-specific]]))
```

Comment:
Same here, let's just leave everything non-essential alone for PR. Can open separate refactor PR. Probably same further down in this PR.

Reply:
I don't think that it's worth it. Opening a PR takes time, and because this cleaning should be done anyway, for me it sounds like a good idea to keep it.

Reply:
@alwx The main reason is to make diffs small, easier to review, merge, avoid regressions and revert later on if necessary if something sneaks in. To me, if refactor is important it is then it is worth doing as an issue, and then it is easy to revert / make sure no regressions creeps in. If this commit later on turns out to introduce regressions we can't just remove it, because then we lose accessibility-id functionality. This is the value of having separate refactor commits. Ivan wrote this earlier: https://hackmd.io/s/rkgqzRWIb